plugins/woocommerce/tests/e2e-pw/tests/merchant/create-shipping-zones.spec.js

```diff
@@ -21,17 +21,6 @@ test.describe( 'WooCommerce Shipping Settings - Add new shipping zone', () => {
 		await api.put( 'settings/general/woocommerce_allowed_countries', {
 			value: 'all',
 		} );
-
-		// Delete all pre-existing shipping zones
-		const { data: zones } = await api.get( 'shipping/zones' );
-
-		const zoneIds = zones
-			.filter( ( { id } ) => id > 0 )
-			.map( ( { id } ) => id );
-
-		for ( const id of zoneIds ) {
-			await api.delete( `shipping/zones/${ id }`, { force: true } );
-		}
```

Comment:
This entire block is now safe to remove because the deletion of all pre-existing shipping zones is now being done on the global setup phase

Reply:
Will this cause a problem with retries though?  Global setup only runs once at the start of the test run.  If this test fails midway through, we could have Shipping Zones already existing, and then it will try to create them again (throwing an error)?

Reply:
Thanks @lanej0 I don't think it would cause an issue because this test has an after all hook that deletes the shipping zones it created earlier. The afterAll hook is triggered after a middle test fails, so the upcoming retry is guaranteed to start without the shipping zones from the previous attempt.

I also think removing this "Delete all shipping zones in the entire site" block would make it safer for other tests that create their own shipping zones to run in parallel with this one.